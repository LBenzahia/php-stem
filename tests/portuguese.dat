a
√°
√†
√£
aacho
aacute
abacaxi
abade
abaet√©
abafar
abafaram
abaixa
abaixe
abaixei
abaixo
abaixou
abalada
abalado
abalaram
abalos
abalou
abalroado
abandona
abandon√°
abandonada
abandonadas
abandonado
abandonados
abandonam
abandonando
abandonar
abandonara
abandonaram
abandonasse
abandono
abandonou
abarrotado
abarrotados
abarrotou
abastada
abastado
abastados
abastecem
abastecer
abastecida
abastecimento
abata
abate
abatedouro
abatem
abatendo
abater
abateram
abates
abateu
abatia
abatidas
abatido
abatidos
abaulado
abb√©
abbey
abc
abcesso
abdel
abdela
abdias
abdica√ß√£o
abdicar
abdomen
abd√¥men
abdominais
abdominal
abecaf√©
abel
aben√ßoadas
aberra√ß√µes
aberta
abertamente
abertas
aberto
abertos
abertura
abic
abicair
abichequer
abifarma
abih
abilio
ab√≠lio
abismo
abissais
abissal
abnega√ß√£o
abocanhar
aboli√ß√£o
abolicionistas
abolido
abolir
abomina
abomina√ß√£o
abomin√°veis
abomin√°vel
abono
aborda
abordada
abordadas
abordado
abordagem
abordagens
abordando
abordar
abordava
abordavam
abor√≠gines
aborrece
aborrecid√≠ssima
aborrecido
aborrecidos
aborrecimentos
aborto
abortos
abouch
abra
abra√£o
abra√ßada
abracadabra
abra√ßando
abra√ßaram
abra√ßo
abra√ßos
abrafe
abrah√£o
abram
abrandar
abrange
abrang√™ncia
abrangendo
abrangente
abranger
abrasador
abrasivo
abre
abrem
abresi
abreu
abrevia√ß√£o
abreviado
abri
abriga
abrigada
abrigam
abrigar
abrigaram
abrigarem
abrigaria
abrigo
abrigou
abril
abrimos
abrindo
abrinq
abrir
abrir√°
abriram
abrir√£o
abrisse
abriu
abrolhos
abrucio
abruptamente
absoluta
absolutamente
absolutas
absoluto
absolutos
absolve
absolvem
absolver
absolveram
absolverem
absolveu
absolvido
absor√ß√£o
abriram
aabsor√√£o
abriram
or√√£o
aor√absolutoor√abssolutoor√rolutooribriram
ornto
absbater
st√™me
abssbanaabsor√√£ter√√£o
tabodo
abranstratas
absoluttratasolutubordadtubors
absolutuborsolutubooolutuboosoluuelite
abujamabra√und√¢o
abranst√™me