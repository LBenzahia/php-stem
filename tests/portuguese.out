a
á
à
ã
aach
aacut
abacax
abad
abaet
abaf
abaf
abaix
abaix
abaix
abaix
abaix
abal
abal
abal
abal
abal
abalro
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abarrot
abarrot
abarrot
abast
abast
abast
abastec
abastec
abastec
abastec
abat
abat
abatedour
abat
abat
abat
abat
abat
abat
abat
abat
abat
abat
abaul
abbé
abbey
abc
abcess
abdel
abdel
abdi
abdic
abdic
abdomen
abdômen
abdomin
abdominal
abecaf
abel
abenço
aberr
abert
abert
abert
abert
abert
abertur
abic
abica
abichequ
abifarm
abih
abih
abihdon
a